src/commands/georadius.json

```diff
@@ -216,25 +216,28 @@
                 {
                     "type": "array",
                     "items": {
-                        "type" : "array",
-                        "prefixItems": {
-                            "description": "name",
-                            "type": "string"
-                        },
+                        "type": "array",
```

Comment:
can you please fix all the georadius commands according to https://github.com/redis/redis/pull/10273#discussion_r1114365742 ?

Reply:
geosearch too

Reply:
@guybe7 Is that correct schema? You can pass `WITHHASH`, WITHCOORD`, `WITHDIST` at the same time (or any combination of these three options). 

```
127.0.0.1:5001> GEOSEARCH Sicily FROMLONLAT 15 37 BYBOX 400 400 km ASC WITHCOORD WITHDIST WITHHASH
1) 1) "Catania"
   2) "56.4413"
   3) (integer) 3479447370796909
   4) 1) (double) 15.08726745843887329
      2) (double) 37.50266842333162032
```
```
127.0.0.1:5001> GEOSEARCH Sicily FROMLONLAT 15 37 BYBOX 400 400 km ASC WITHCOORD WITHDIST 
1) 1) "Catania"
   2) "56.4413"
   3) 1) (double) 15.08726745843887329
      2) (double) 37.50266842333162032
```

Reply:
🤦‍♂️ 
does the order of args change the order of the array? i sure hope not
i.e. `GEOSEARCH .. WITHCOORD WITHDIST` vs. `GEOSEARCH .. WITHDIST WITHCOORD`

Reply:
regardless, you're right, but i think that it's better to have "items" as an array with one element describing the member name (so that the validator will validate it's the first element) and the rest (oneOf of all the possible outcomes) should be in "additionalItems"

it's still not perfect but it's better than what we have name (the difference is validating the member name is first)

Reply:
No, the order of the args does not change the array.

Reply:
btw according to your example the outcome of WITHDIST is a string, not a number

Reply:
good catch. I'll fix it.